lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_converter.cpp

```diff
@@ -145,21 +146,17 @@ void S1apStateConverter::proto_to_enb(const oai::EnbDescription& proto,
   enb->ran_cp_ipaddr_sz = proto.ran_cp_ipaddr_sz();
 
   // load ues
-  hashtable_rc_t ht_rc;
-  auto ht_name = bfromcstr("s1ap_ue_coll");
-
-  hashtable_uint64_ts_init(&enb->ue_id_coll, mme_config.max_ues, nullptr,
-                           ht_name);
-  bdestroy(ht_name);
+  proto_map_rc_t rc = {PROTO_MAP_OK};
+  enb->ue_id_coll.map = new google::protobuf::Map<uint32_t, uint64_t>();
```

Comment:
The proto map is using a uint64 key. Can we use the same width here?

Reply:
We discussed that, since mme_ue_s1ap_if type uint32 and shall change the datatype in proto file.